buzzmobile/tests/rostest_utils.py

```diff
@@ -145,9 +145,13 @@ def __init__(self, topic, msg_type):
         self.msg_type = msg_type
         self.received = False
 
-    def wait_for_message(self):
-        while not self.received:
+    def wait_for_message(self, timeout=10):
+        elapsed = 0
+        while not self.received and elapsed < timeout:
             time.sleep(.1)
+            elapsed += .1
+        if elapsed >= timeout:
+            raise TimeoutError('Test timed out')
```

Comment:
How about 'Timed out waiting for message'? More descriptive of where it timed out.